packages/graphiql/README.md

```diff
@@ -201,7 +201,9 @@ GraphiQL supports customization in UI and behavior by accepting React props and
 
 - `headerEditorEnabled`: an optional boolean which enables the header editor when `true`. Defaults to `false`.
 
-- `shouldPersistHeaders`: an optional boolean which enables to persist headers to storage when `true`. Defaults to `false`
+- `shouldPersistHeaders`: an optional boolean which enables to persist headers to storage when `true`. Defaults to `false`.
+
+- `runMultipleQueries`: an optional boolean which enables to run multiple queries when `true`. The actual response of each query is returned as `data` object within the `yourQueryNameData` object. Defaults to `false`.
```

Comment:
can we link this to the spec document? is this what the spec document calls this feature?

Reply:
The feature name is as suggested in the issue. Also I could not find the spec document for graphiql, it would be great if you could share the link!

Reply:
i mean the spec document for this language feature. in the language spec. can you link to it in the readme here?